### PR TITLE
Swap latitude and longitude in geo points.

### PR DESCRIPTION
**Pull request by BillWhite**
Many vendors list points as (longitude, latitude).  This matches
our normal (x, y) intuition.  This is not universal, but it seems
to be more common than (latitude, longitude).  So, we have switched
these.

https://issues.voltdb.com/browse/ENG-9493

### CODE REVIEW

**Comment by hhakimi53** [comment]
Errors messages in throwInvalidWktPoint(), throwInvalidWktPoly() (src/ee/expressions/geofunctions.cpp) will have to updated. Currently it's formed based on that user will provide latitude, longitude for points and are:
"expected input of the form 'POINT(<lat> <lng>)" for pointfromtext  and "Expected input of the form 'POLYGON((<lat> <lng>, ...), ...)' " for polygonfromtext
to 
"expected input of the form 'POINT(<lng> <lat>)" and "expected input of the form 'POINT(<lat> <lng>)". 

This changes will also require changes to negative junit test cases.

File: src/frontend/org/voltdb/types/GeographyValue.java [hunk 1]
```diff
@@ -470,13 +470,13 @@ private static void diagnoseLoop(List<XYZPoint> loop, String excpMsgPrf) throws
                         throw new IllegalArgumentException(msgPrefix + "missing opening parenthesis");
                     }
 
-                    double lat = tokenizer.nval;
+                    double lng = tokenizer.nval;
                     token = tokenizer.nextToken();
                     if (token != StreamTokenizer.TT_NUMBER) {
                         throw new IllegalArgumentException(msgPrefix + "missing longitude in lat long pair");
```
Comment (hhakimi53) [inline src/frontend/org/voltdb/types/GeographyValue.java]: Message will have to be updated to: "missing latitude in long lat pair"

**Comment by hhakimi53** [comment]
Overall changes looks fine besides the cosmetic message changes for lat/long pair in logic and tests.

**Comment by BillWhite** [comment]
So, this is mostly blue in Jenkins.  The only junit failure is one of the usual ones.

**Comment by BillWhite** [comment]
Squashed.

**Pull request closed without merging**